data/terminator.metainfo.xml

```diff
@@ -2,47 +2,47 @@
- <project_license>GPL-2.0 only</project_license>
- <_name>Terminator</_name>
- <_summary>Multiple terminals in one window</_summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0 only</project_license>
+ <name translatable="yes">Terminator</name>
+ <summary translatable="yes">Multiple terminals in one window</summary>
  <description>
-  <_p>
+  <p translatable="yes">
    The robot future of terminals
-  </_p>
-  <_p>
+  </p>
+  <p translatable="yes">
    A power-user tool for arranging terminals. It is inspired by programs such as
    gnome-multi-term, quadkonsole, etc. in that the main focus is arranging terminals
    in grids (tabs is the most common default method, which Terminator also supports).
-  </_p>
-  <_p>
+  </p>
+  <p translatable="yes">
    Much of the behavior of Terminator is based on GNOME Terminal, and we are adding
    more features from that as time goes by, but we also want to extend out in different
    directions with useful features for sysadmins and other users.
-  </_p>
-  <_p>Some highlights:</_p>
+  </p>
+  <p translatable="yes">Some highlights:</p>
   <ul>
-   <_li>Arrange terminals in a grid</_li>
-   <_li>Tabs</_li>
-   <_li>Drag and drop re-ordering of terminals</_li>
-   <_li>Lots of keyboard shortcuts</_li>
-   <_li>Save multiple layouts and profiles via GUI preferences editor</_li>
-   <_li>Simultaneous typing to arbitrary groups of terminals</_li>
+   <li translatable="yes">Arrange terminals in a grid</li>
+   <li translatable="yes">Tabs</li>
+   <li translatable="yes">Drag and drop re-ordering of terminals</li>
+   <li translatable="yes">Lots of keyboard shortcuts</li>
+   <li translatable="yes">Save multiple layouts and profiles via GUI preferences editor</li>
+   <li translatable="yes">Simultaneous typing to arbitrary groups of terminals</li>
   </ul>
-  <_p>And lots more...</_p>
+  <p translatable="yes">And lots more...</p>
  </description>
  <screenshots>
   <screenshot type="default">
    <image>http://4.bp.blogspot.com/-xt4Tja1TMQ0/Vdemmf8wYSI/AAAAAAAAA9A/uROTre0PMls/s1600/terminator_main_basic.png</image>
-   <_caption>The main window showing the application in action</_caption>
+   <caption translatable="yes">The main window showing the application in action</caption>
   </screenshot>
   <screenshot>
    <image>http://4.bp.blogspot.com/-rRxALSpEEZw/Vdeu58JgpnI/AAAAAAAAA9o/XewWKJ5HNo4/s1600/terminator_main_complex.png</image>
-   <_caption>Getting a little crazy with the terminals</_caption>
+   <caption translatable="yes">Getting a little crazy with the terminals</caption>
   </screenshot>
   <screenshot>
-   <image>http://2.bp.blogspot.com/-t_8oRyMXUls/VdemmRVnZnI/AAAAAAAAA88/rHIr8L1X7Ho/s1600/terminator_prefs_global.png</image>
-   <_caption>The preferences window where you can change the defaults</_caption>
+   <image>http://2.bp.blogspot.com/-t_8oRyMXUls/VdemmRVnZnI/AAAAAAAAA88/rHIr8L1X7Ho/s1600/terminatorp translatable="yes"refs_global.png</image>
```

Comment:
The `<img>` got corrupted by a replacement

Reply:
What Jakub said :smile:

Reply:
```suggestion
   <image>http://2.bp.blogspot.com/-t_8oRyMXUls/VdemmRVnZnI/AAAAAAAAA88/rHIr8L1X7Ho/s1600/terminatorprefs_global.png</image>
```